src/components/organisms/FamilyConceptPicker.tsx

```diff
@@ -0,0 +1,199 @@
+import { TextSearch } from "lucide-react";
+import { NextRouter, useRouter } from "next/router";
+import { useContext, useEffect, useRef, useState } from "react";
+
+import { Accordian } from "@/components/accordian/Accordian";
+import { Badge } from "@/components/atoms/label/Badge";
+import { Select } from "@/components/atoms/select/Select";
+import { InputCheck } from "@/components/forms/Checkbox";
+import { NewFeatureCard } from "@/components/molecules/newFeatures/NewFeatureCard";
+import { NEW_FEATURES } from "@/constants/newFeatures";
+import { QUERY_PARAMS } from "@/constants/queryParams";
+import { NewFeatureContext } from "@/context/NewFeatureContext";
+import { TConcept } from "@/types";
+import { CleanRouterQuery } from "@/utils/cleanRouterQuery";
+import { groupByRootConcept } from "@/utils/conceptsGroupedbyRootConcept";
+import { firstCase } from "@/utils/text";
+
+import { ExternalLink } from "../ExternalLink";
+
+interface IProps {
+  concepts: TConcept[];
+  containerClasses?: string;
+  showBadge?: boolean;
+  showSearch?: boolean;
+  startingSort?: TSort;
+  title: string;
+}
+
+const SORT_OPTIONS = ["A-Z", "Grouped"] as const;
+
+type TSort = (typeof SORT_OPTIONS)[number];
+
+const isSelected = (queryValue: string | string[] | undefined, option: string) => {
+  if (!queryValue) {
+    return false;
+  }
+  if (typeof queryValue === "string") {
+    return queryValue === option;
+  } else {
+    return queryValue.includes(option);
+  }
+};
+
+const conceptsSorter = (a: TConcept, b: TConcept, sort: TSort) => {
+  if (sort === "A-Z") {
+    return a.preferred_label.localeCompare(b.preferred_label);
+  } else if (sort === "Grouped") {
+    return a.recursive_subconcept_of[0].localeCompare(b.recursive_subconcept_of[0]);
+  }
+  return 0;
+};
+
+const filterConcepts = (concepts: TConcept[], search: string) => {
+  return concepts.filter(
+    (concept) =>
+      concept.preferred_label.toLowerCase().includes(search.toLowerCase()) ||
+      concept.alternative_labels.some((label) => label.toLowerCase().includes(search.toLowerCase()))
+  );
+};
+
+const onConceptChange = (router: NextRouter, concept: TConcept) => {
+  const query = CleanRouterQuery({ ...router.query });
+  // Retain any dynamic ids in the query (e.g. document page)
+  if (router.query.id) {
+    query["id"] = router.query.id;
+  }
+  let selectedConcepts = query[QUERY_PARAMS.concept_name] ? [query[QUERY_PARAMS.concept_name]].flat() : [];
+
+  if (selectedConcepts.includes(concept.preferred_label)) {
+    selectedConcepts = selectedConcepts.filter((c) => c !== concept.preferred_label);
+  } else {
+    selectedConcepts = [...selectedConcepts, concept.preferred_label];
+  }
+
+  query[QUERY_PARAMS.concept_name] = selectedConcepts;
+
+  router.push({ query: query }, undefined, { shallow: true });
+};
+
+export const FamilyConceptPicker = ({
```

Comment:
There's probably some abstraction we can make here - but it would be good to compare once we get the GeographiesPicker in from @odrakes-cpr and @annaCPR to have another example to abstract to.